data/mt/retired/Mary-Ann-Dunwell-3e82bb94-a815-40d2-b125-b9884517b1c3.yml

```diff
@@ -3,18 +3,24 @@ name: Mary Ann Dunwell
 given_name: Mary Ann
 family_name: Dunwell
 gender: Female
-email: maryann.dunwell@mtleg.gov
+email: MaryAnn.Dunwell@legmt.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person was in the house from 2015 until 01/02/2023. On that date they start in the senate